src/main/java/student/Student.java

```diff
@@ -1,32 +1,29 @@
 package student;
 
+import java.util.ArrayList;
+
 public class Student {
-    private String name;
-    private int id;
+    private String id;
     private float gpa;
-    private int[] uniPreferences = new int[3];
+    private ArrayList<Integer> uniPreferences;
     private boolean isSuccessfullyAllocated = false;
+    private int allocatedUniversity = -1;

```

Comment:
do you think we should set allocatedUniversity to be a university type? In the future will it be more convenient for our code?

Reply:
Do you guys mean an enum?